app/forms/network-interface-edit.tsx

```diff
@@ -65,6 +112,76 @@ export function EditNetworkInterfaceForm({
     >
       <NameField name="name" control={form.control} />
       <DescriptionField name="description" control={form.control} />
+      <FormDivider />
+
+      <div className="flex flex-col gap-3">
+        {/* We have to blow this up instead of using TextField to get better text styling on the label */}
+        <div className="mt-2">
+          <label id="transitIps-label" htmlFor="transitIp" className="text-sans-lg">
+            Transit IPs
+          </label>
+          <TextInputHint id="transitIps-help-text" className="mb-2">
+            Enter an IPv4 or IPv6 address, or{' '}
```

Comment:
Love including a docs link here. No need to frame it as a choice between two things.

```suggestion
            Enter an IPv4 or IPv6 address. {' '}
```

Not sure either about the icon. I’m surprised we haven’t run into this question before. I lean slightly against, I think.